Backend/src/model/section.py

```diff
@@ -4,10 +4,10 @@
 
 @dataclass
 class Section:
-    courseCode: str
-    sectionId: str 
+    course_code: str
```

Comment:
Is the course_code field the same value as the "courses" data class's code field?

Reply:
Yes